src/auth/auth-api-request.ts

```diff
@@ -236,6 +237,21 @@ class TenantAwareAuthResourceUrlBuilder extends AuthResourceUrlBuilder {
   }
 }
 
+/**
+ * Auth-specific HTTP client which uses the special "owner" token
+ * when communicating with the Auth Emulator.
+ */
+class AuthHttpClient extends AuthorizedHttpClient {
+
+  protected getToken(): Promise<string> {
+    if (useEmulator()) {
+      return Promise.resolve('owner');
```

Comment:
Please add a unit test for this. Also why did you call it `owner`?

Reply:
It's a convention for Firestore, RTDB, and now Auth Emulators to treat the access token `owner` as if it belonged to a service account that had IAM Role "Project Owner" (granting all permissions), hence the naming. This is documented at http://go/emulator-suite-admin-sdk.